src/components/ec-currency-input/ec-currency-input.spec.js

```diff
@@ -45,6 +51,47 @@ describe('EcCurrencyInput', () => {
     expect(wrapper.element).toMatchSnapshot();
   });
 
+  describe('when the bottom note is defined', () => {
```

Comment:
capitalize first part of the spec statement

Reply:
Hey Carlos, I'm not totally sure about this, since it seems that that first capitalized word is something we only do (so far) in integration tests, not in unit ones (neither in Chameleon nor in EBO). If we want to start capitalizing it is fine with me, but I just wanted to double-check it.

Reply:
you're correct, in that case